vscode-wpilib/resources/webviews/gradle2025import.html

```diff
@@ -2,111 +2,186 @@
-      this will do some unexpected things during build. In addition, not all vendor libraries
-      support desktop. This option can be set with the command "WPILib: Set Desktop Support" at any
-      time.
-    </div>
-    <br />
-    <div id="romidiv">
-      <b>Import Romi Project</b>
-      <input id="romiCB" type="checkbox" />
-      <br />
-      Check this if the project being imported is a romi project. This is not autodetected, and must
-      be selected manually.
-    </div>
-    <br />
-    <div id="xrpdiv">
-      <b>Import XRP Project</b>
-      <input id="xrpCB" type="checkbox" />
-      <br />
-      Check this if the project being imported is an XRP project. This is not autodetected, and must
-      be selected manually.
+    <div class="project-container">
+      <img src="replaceresource/resources/wpilib-generic.svg" height="75" />
+
+      <h1 class="project-title">Import WPILib 2025 Project</h1>
+
+      <div class="wizard-progress">
+        <div class="progress-step active" data-step="1">Select Source</div>
+        <div class="progress-step" data-step="2">Configure Project</div>
+        <div class="progress-step" data-step="3">Review & Import</div>
+      </div>
+
+      <!-- Step 1: Select Source Project -->
+      <div class="wizard-step active" id="step-1">
+        <div class="step-header">
+          <h2>Step 1: Select Source Project</h2>
+          <p>Select the build.gradle file of your 2025 WPILib project that you want to import.</p>
+        </div>
+
+        <div class="project-row">
+          <div class="project-label"><b>Gradle Project</b></div>
+          <span>Select the build.gradle file of the gradle project to import.</span>
+        </div>
+
+        <div class="project-row">
+          <input id="gradle2025Input" type="text" class="vscode-textfield" readonly />
+        </div>
+
+        <div class="project-row">
+          <button id="gradle2025SelectButton" class="vscode-button">Select Source Project</button>
+        </div>
+
+        <div class="wizard-navigation">
+          <div></div>
+          <!-- Empty div for alignment -->
+          <button id="next-to-step-2" class="vscode-button next-button" disabled>Next</button>
+        </div>
+      </div>
+
+      <!-- Step 2: Configure Project -->
+      <div class="wizard-step" id="step-2">
+        <div class="step-header">
+          <h2>Step 2: Configure Project</h2>
+          <p>Configure the new project's location and settings.</p>
+        </div>
+
+        <div class="project-row">
+          <div id="projectfolderdiv" class="project-label"><b>Base Folder</b></div>
+          <span>Select a base folder to place the new project into.</span>
+        </div>
+
+        <div class="project-row">
+          <input id="projectFolder" type="text" class="vscode-textfield" readonly />
+        </div>
+
+        <div class="project-row">
+          <button id="projectSelectButton" class="vscode-button">Select Destination Folder</button>
```

Comment:
This is called Destination folder, but all the other text talks about base folder